meta-resin-common/recipes-support/hostapp-update-hooks/files/0-bootfiles

```diff
@@ -90,16 +90,47 @@ deploy() {
 # MAIN
 #
 
-export DOCKER_HOST="unix:///var/run/docker-host.sock"
+export DOCKER_HOST="unix:///var/run/balena-host.sock"
 
 if [ -z "$container_id" ]; then
 	echo "[ERROR] Container ID provided as an argument is needed."
 	exit 1
 fi
 
-# Deploy all files in the bootfiles list
 bootfiles_list_new="/tmp/$(basename $bootfiles_list).new"
-docker cp "$container_id:$bootfiles_list" "$bootfiles_list_new"
+balena cp "$container_id:$bootfiles_list" "$bootfiles_list_new"
```

Comment:
Why do we need a list here instead of enumerating all the files in `/resin-boot`?

Reply:
This is a relic now given that I switched to running the hooks in a container.

Reply:
Will this relic be removed in this PR or in a follow-up PR?

Reply:
It is part of this PR. Go through all the commits. It suffered a couple of iterations.

Reply:
Can they be squashed? Makes more sense to push a PR with the final form rather than having commits that amend earlier commits in the same PR. Easier for review also

Reply:
I'm not currently planning to squash this PR.